cypress/integration/project_creation.spec.js

```diff
@@ -130,18 +130,17 @@ context("project creation", () => {
       cy.get("body").type("{esc}", { force: true });
     });
 
-    context("name", () => {
+    context.only("name", () => {
```

Comment:
```suggestion
    context("name", () => {
```

Our eslint plugin has failed us :( This should have failed on CI.